docs/lib/routers.md

```diff
@@ -0,0 +1,69 @@
+# Associative Resource Router
+
+django-ansible-base provides an `AssociationResourceRouter` which will auto-construct /associate and /disassociate endpoints for related ManyToMany fields for your models.
```

Comment:
Based on your other comments, do we need to update this statement to include reverse ForeignKey (many-to-one) relationships as well?

Reply:
Good callout, I need to update that file.